new_symbolic.jl

```diff
@@ -0,0 +1,163 @@
+import YAML
```

Comment:
What is the difference between this file and the src/model.jl?

It seems like many things are repeated between the two files. and that this file is not included in the package